### PR TITLE
[New Rule] User Added as Owner for Azure Application

### LINKED ISSUE #112: [New Rule] User Added as Owner for Azure Application
## Description

Identifies when a user is added as an owner for an Azure application. An adversary may add a user account as an owner for an Azure application in order to grant additional permissions and modify the application's configuration using another account.

```
 event.module:azure and event.dataset:azure.auditlogs and event.category:AuditLogs and azure.auditlogs.operation_name:"Add owner to application" and event.outcome: "Success"
```

## Required Info

- **Eventing Sources:**

`filebeat-*`

- **Target Operating Systems:**

- **Platforms**

Azure

- **Target ECS Version:** `1.5.0`
- **New fields required in ECS for this?** No
- **Related issues or PRs** None

## Optional Info
- **References:**

## Example Data

image

### PR DESCRIPTION
**Pull request by threat-punter**
## Issues

Resolves #112 

## Summary
Identifies when a user is added as an owner for an Azure application. An adversary may add a user account as an owner for an Azure application in order to grant additional permissions and modify the application's configuration using another account.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/azure/defense_evasion_user_added_as_owner_for_azure_application.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+name = "User Added as Owner for Azure Application"
+note = "The Azure Filebeat module must be enabled to use this rule."
+risk_score = 21
+rule_id = "774f5e28-7b75-4a58-b94e-41bf060fdd86"
+severity = "low"
+tags = ["Elastic", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+type = "query"
+
+query = '''
+event.module:azure and event.dataset:azure.auditlogs and event.category:AuditLogs and azure.auditlogs.operation_name:"Add owner to application" and event.outcome: "Success"
```
Comment (brokensound77) [inline rules/azure/defense_evasion_user_added_as_owner_for_azure_application.toml:27]: nit:

```suggestion
event.module:azure and event.dataset:azure.auditlogs and event.category:AuditLogs and azure.auditlogs.operation_name:"Add owner to application" and event.outcome:Success
```

Comment (threat-punter) [reply rules/azure/defense_evasion_user_added_as_owner_for_azure_application.toml:27]: Thanks!

**Review by Samirbous — APPROVED** [review]

File: rules/azure/defense_evasion_user_added_as_owner_for_azure_application.toml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+[metadata]
+creation_date = "2020/08/20"
+ecs_version = ["1.5.0"]
```
Comment (peasead) [inline rules/azure/defense_evasion_user_added_as_owner_for_azure_application.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

**Comment by threat-punter** [comment]
> Account Manipulation?

Good call. Just added that to the rule. 🥳

**Pull request merged**